apps/dashboard/app/lib/account_cache.rb

```diff
@@ -169,4 +167,21 @@ def account_allowed?(queue, account_name)
       queue.allow_accounts.any? { |account| account == account_name }
     end
   end
+
+  def disabled_account_data(disabled_accounts)
+    counter = 0
+    disabled_accounts.map do |account|
+      counter += 1
+      # check if account contains anything other than digits and lowercase letters
+      if !!(account =~ /\A[a-z0-9]+\z/)
```

Comment:
Can this be `/\A[a-z0-9]+\z/.match?(account)` instead? For whatever reason the `!!` is throwing me off. I know it's to account cast `nils`, it's just double negative logic always makes me do a double take.